Ethinylestradiol/Ethinylestradiol-Report.md

```diff
@@ -0,0 +1,419 @@
+
+**Orme 1991** Orme M, Back DJ, Ward S, Green S. The pharmacokinetics of ethynylestradiol in the presence and absence of gestodene and desogestrel. *Contraception*. 1991;43(4):305-316.
+
+**PK-Sim Ontogeny Database Version 7.3** (https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/blob/38cf71b384cfc25cfa0ce4d2f3addfd32757e13b/PK-Sim%20Ontogeny%20Database%20Version%207.3.pdf)
+
+**Sidhu 2006** Sidhu J, Job S, Singh S, Philipson R. The pharmacokinetic and pharmacodynamic consequences of the co-administration of lamotrigine and a combined oral contraceptive in healthy female subjects. *Br J Clin Pharmacol*. 2006;61(2):191-199.
+
+**SmPC Namuscla ** SmPC Namuscla 167 mg hard capsules, 2019, website medicines.org.uk/emc/product/9838/smpc
+
+**Stanczyk 1983** Stanczyk FZ, Mroszczak EJ, Ling T, et al. Plasma levels and pharmacokinetics of norethindrone and ethinylestradiol administered in solution and as tablets to women. *Contraception*. 1983;28(3):241-251.
+
+**Stanczyk 2013** Stanczyk FZ, Archer DF, Bhavnani BR. Ethinyl estradiol and 17beta-estradiol in combined oral contraceptives: pharmacokinetics, pharmacodynamics and risk assessment. *Contraception*. 2013;87(6):706-727.
+
+**Stockis 2014** Stockis A, Watanabe S, Fauchoux N. Interaction between brivaracetam (100 mg/day) and a combination oral contraceptive: A randomized, double-blind, placebo-controlled study. *Epilepsia*. 2014;55(3):27-31.
+
+**Study c13608215-02** Study c13608215. A study to investigate the pharmacokinetic drug-drug interaction following oral administration of ethinylestradiol/levonorgestrel (Microgynon®) and BI 409306 in healthy Korean premenopausal female subjects (an open-label, two-period, fixed-sequence study).” Boehringer Ingelheim Pharma, 06-Aug-2018.
+
+**Timmer 2000** Timmer C, Mulders T. Pharmacokinetics of etonogestrel and ethinylestradiol released from a combined vaginal ring. *Clin Pharmacokinet*. 2000;39(3):233-242.
+
+**Wang 2004** Wang, B., Sanchez, R.I., Franklin, R.B., Evans, D.C.,Huskey, S.E. The involvement of CYP3A4 and CYP2C9 in the metabolism of 17 alpha-ethinylestradiol. *Drug Metab. Dispos*. 32, 1209–1212 (2004).
+
+**Wiesinger 2015**  Wiesinger, H. et al. Pharmacokinetic interaction between the CYP3A4 inhibitor ketoconazole and the hormone drospirenone in combination with ethinylestradiol or estradiol. *Br. J. Clin. Pharmacol*. 80, 1399–1410 (2015).
+
+**Willmann 2007** Willmann S, Höhn K, Edginton A, Sevestre M, Solodenko J, Weiss W, Lippert J, Schmitt W. Development of a physiology-based whole-body population model for assessing the influence of individual variability on the pharmacokinetics of drugs. *J Pharmacokinet Pharmacodyn* 2007, 34(3): 401-431.
+
+**Zanaflex prescribing information**  Zanaflex prescribing information. Website: accessdata.fda.gov/drugsatfda_docs/label/2006/020397s021,021447s002lbl.pdf, 2006, Acorda Therapeutics Inc
```

Comment:
accessdata.fda.gov/drugsatfda_docs/label/2006/020397s021,021447s002lbl.pdf
Link not properly rendered. Use this instead?
https://www.accessdata.fda.gov/drugsatfda_docs/label/2006/020397s021,021447s002lbl.pdf

Reply:
Fixed.